cypress/commons/actions/generic/Scenarios.js

```diff
@@ -51,15 +52,25 @@ function switchToScenarioView() {
 }
 
 // Select the scenario with the provided name and id
-function selectScenario(scenarioName, scenarioId) {
+function selectScenario(scenarioName, scenarioId, selectTimeout) {
```

Comment:
Can we have a default value for the `selectTimeout` parameter ? This argument does not seem to be mandatory when writing the tests, and we want to keep the tests body as simple as possible.

Reply:
Same as above, I put default timeout in getter function :
`function getScenarioSelector(timeout = 5) {
  return cy.get(GENERIC_SELECTORS.scenario.selectInput, { timeout: timeout * 1000 });
}`